src/collective/solr/manager.py

```diff
@@ -1,67 +1,23 @@
 # -*- coding: utf-8 -*-
-from collective.solr.interfaces import ISolrConnectionConfig
 from collective.solr.interfaces import ISolrConnectionManager
 from collective.solr.interfaces import IZCMLSolrConnectionConfig
 from collective.solr.local import getLocal
 from collective.solr.local import setLocal
 from collective.solr.solr import SolrConnection
+from collective.solr.utils import getConfig
+from collective.solr.utils import isActive
 from httplib import CannotSendRequest
 from httplib import ResponseNotReady
 from logging import getLogger
-from persistent import Persistent
 from socket import error
-from zope.component import getUtility
 from zope.component import queryUtility
 from zope.interface import implements
+from plone.api.portal import get_registry_record
 
 logger = getLogger('collective.solr.manager')
 marker = object()
 
 
-class BaseSolrConnectionConfig(object):
```

Comment:
@tisto Configuration moved to the registry. There is no need for a separate way to config solr and the registry is relative to a Plone instance which should allow multiple Plone instances in on Zope. Did I miss something?